httpclient5/src/main/java/org/apache/hc/client5/http/entity/mime/MultipartEntityBuilder.java

```diff
@@ -204,6 +214,20 @@ public MultipartEntityBuilder addBinaryBody(final String name, final InputStream
         return addBinaryBody(name, stream, ContentType.DEFAULT_BINARY, null);
     }
 
+    /**
+     * Returns the fixed default boundary value.
+     */
+    private String getFixedBoundary() {
+        return "httpclient_boundary_7k9p2m4x8n5j3q6t1r0vwyzabcdefghi";
+    }
+
+    /**
+     * Generates a random boundary using UUID.
+     */
+    private String getRandomBoundary() {
+        return UUID.randomUUID().toString();
```

Comment:
@arturobernalg Can we add `httpclient_boundary_` in front of it?

Reply:
@ok2c  done